internal/repository/user_email_repository.go

```diff
@@ -0,0 +1,112 @@
+package repository
+
+import (
+	"context"
+	"fmt"
+
+	sq "github.com/Masterminds/squirrel"
+	"github.com/jackc/pgx/v5"
+	"github.com/jackc/pgx/v5/pgxpool"
+	"github.com/joshsoftware/profile_builder_backend_go/internal/pkg/constants"
+	"github.com/joshsoftware/profile_builder_backend_go/internal/pkg/helpers"
+	"github.com/joshsoftware/profile_builder_backend_go/internal/pkg/specs"
+	"go.uber.org/zap"
+)
+
+type EmailStore struct {
+	db *pgxpool.Pool
+}
+
+type EmailStorer interface {
+	GetInvitations(ctx context.Context, profileID int, tx pgx.Tx) (specs.InvitationResponse, error)
+	CreateInvitation(ctx context.Context, invitation Invitations, tx pgx.Tx) error
+	UpdateProfileCompleteStatus(ctx context.Context, profileID int, updateReq UpadateRequest, tx pgx.Tx) error
+}
+
+func NewUserEmailRepo(db *pgxpool.Pool) EmailStorer {
+	return &EmailStore{
+		db: db,
+	}
+}
+
+func (userStore *EmailStore) GetInvitations(ctx context.Context, profileID int, tx pgx.Tx) (specs.InvitationResponse, error) {
+	queryBuilder := psql.Select(constants.RequestInvitationColumns...).From("invitations").Where(sq.And{sq.Eq{"profile_id": profileID}, sq.Eq{"is_profile_complete": 0}})
```

Comment:
const ->invitations

Reply:
we already discussed. 
send `sq.Eq{"is_profile_complete": 0}}` in method signature as filter

Reply:
done